src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectRevisionController.java

```diff
@@ -190,7 +194,7 @@ public CatalogObjectMetadata create(
         }
 
         CatalogObjectMetadata catalogObjectMetadata = catalogObjectService.restore(bucketName, name, commitTimeRaw);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "catalogObjectMetadata". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)

Reply:
I agree we can return immediately the result